### ISSUE TITLE
draft github classroom

### ISSUE THREAD

**Issue by nicholasjhorton** [hglanz/DS-Blog-Ex#28]
to build on github intro and github in R entries (shameless plug for FIksel and Hardin)

**Comment by hardin47** [comment]
draft of github classroom is now posted.  looking forward to hearing feedback from @hglanz and @nicholasjhorton .  I also sent it to jacob to ask him for feedback.

**Comment by nicholasjhorton** [comment]
Potential to add in a plug for gfclass?  (Or note it?)


Register Now!Abstract DetailsActivity Number:514 - Teaching Data Science: R, Git, and the Undergraduate CurriculumType:ContributedDate/Time:Wednesday, July 31, 2019 : 10:30 AM to 12:20 PMSponsor:Section on Statistics and Data Science EducationAbstract #306367Title:ghclass: an R package for managing classes with GitHubAuthor(s):Colin Rundel* Companies:Duke University Keywords:github; reproducibility; data science; computing; classroom management; RAbstract:In this talk we will present the details of the ghclass package which we have developed as a tool for managing statistical and data science courses with a computation component. The package is designed to automate the process of distributing and collecting assignments via git repositories hosted on GitHub. As part of this talk will discuss best practices for how these tools can be deployed in a variety of classroom settings from first year introductory courses through graduate level courses. Finally we will discuss how these tools fit into the larger context of modern statistical / data science pedagogy with a particular emphasis on the role of reproducibility in training new researchers. Authors who are presenting talks have a * after their name.Back to the full JSM 2019 program |   |   | Register Now! | Activity Number:514 - Teaching Data Science: R, Git, and the Undergraduate CurriculumType:ContributedDate/Time:Wednesday, July 31, 2019 : 10:30 AM to 12:20 PMSponsor:Section on Statistics and Data Science EducationAbstract #306367Title:ghclass: an R package for managing classes with GitHubAuthor(s):Colin Rundel* Companies:Duke University Keywords:github; reproducibility; data science; computing; classroom management; RAbstract:In this talk we will present the details of the ghclass package which we have developed as a tool for managing statistical and data science courses with a computation component. The package is designed to automate the process of distributing and collecting assignments via git repositories hosted on GitHub. As part of this talk will discuss best practices for how these tools can be deployed in a variety of classroom settings from first year introductory courses through graduate level courses. Finally we will discuss how these tools fit into the larger context of modern statistical / data science pedagogy with a particular emphasis on the role of reproducibility in training new researchers. Authors who are presenting talks have a * after their name.Back to the full JSM 2019 program | Activity Number: | 514 - Teaching Data Science: R, Git, and the Undergraduate Curriculum | Type: | Contributed | Date/Time: | Wednesday, July 31, 2019 : 10:30 AM to 12:20 PM | Sponsor: | Section on Statistics and Data Science Education | Abstract #306367 |   | Title: | ghclass: an R package for managing classes with GitHub | Author(s): | Colin Rundel* | Companies: | Duke University | Keywords: | github; reproducibility; data science; computing; classroom management; R | Abstract: | In this talk we will present the details of the ghclass package which we have developed as a tool for managing statistical and data science courses with a computation component. The package is designed to automate the process of distributing and collecting assignments via git repositories hosted on GitHub. As part of this talk will discuss best practices for how these tools can be deployed in a variety of classroom settings from first year introductory courses through graduate level courses. Finally we will discuss how these tools fit into the larger context of modern statistical / data science pedagogy with a particular emphasis on the role of reproducibility in training new researchers.
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
  |   | Register Now!
Activity Number:514 - Teaching Data Science: R, Git, and the Undergraduate CurriculumType:ContributedDate/Time:Wednesday, July 31, 2019 : 10:30 AM to 12:20 PMSponsor:Section on Statistics and Data Science EducationAbstract #306367Title:ghclass: an R package for managing classes with GitHubAuthor(s):Colin Rundel* Companies:Duke University Keywords:github; reproducibility; data science; computing; classroom management; RAbstract:In this talk we will present the details of the ghclass package which we have developed as a tool for managing statistical and data science courses with a computation component. The package is designed to automate the process of distributing and collecting assignments via git repositories hosted on GitHub. As part of this talk will discuss best practices for how these tools can be deployed in a variety of classroom settings from first year introductory courses through graduate level courses. Finally we will discuss how these tools fit into the larger context of modern statistical / data science pedagogy with a particular emphasis on the role of reproducibility in training new researchers. Authors who are presenting talks have a * after their name.Back to the full JSM 2019 program | Activity Number: | 514 - Teaching Data Science: R, Git, and the Undergraduate Curriculum | Type: | Contributed | Date/Time: | Wednesday, July 31, 2019 : 10:30 AM to 12:20 PM | Sponsor: | Section on Statistics and Data Science Education | Abstract #306367 |   | Title: | ghclass: an R package for managing classes with GitHub | Author(s): | Colin Rundel* | Companies: | Duke University | Keywords: | github; reproducibility; data science; computing; classroom management; R | Abstract: | In this talk we will present the details of the ghclass package which we have developed as a tool for managing statistical and data science courses with a computation component. The package is designed to automate the process of distributing and collecting assignments via git repositories hosted on GitHub. As part of this talk will discuss best practices for how these tools can be deployed in a variety of classroom settings from first year introductory courses through graduate level courses. Finally we will discuss how these tools fit into the larger context of modern statistical / data science pedagogy with a particular emphasis on the role of reproducibility in training new researchers.
Activity Number: | 514 - Teaching Data Science: R, Git, and the Undergraduate Curriculum
Type: | Contributed
Date/Time: | Wednesday, July 31, 2019 : 10:30 AM to 12:20 PM
Sponsor: | Section on Statistics and Data Science Education
Abstract #306367 |  
Title: | ghclass: an R package for managing classes with GitHub
Author(s): | Colin Rundel*
Companies: | Duke University
Keywords: | github; reproducibility; data science; computing; classroom management; R
Abstract: | In this talk we will present the details of the ghclass package which we have developed as a tool for managing statistical and data science courses with a computation component. The package is designed to automate the process of distributing and collecting assignments via git repositories hosted on GitHub. As part of this talk will discuss best practices for how these tools can be deployed in a variety of classroom settings from first year introductory courses through graduate level courses. Finally we will discuss how these tools fit into the larger context of modern statistical / data science pedagogy with a particular emphasis on the role of reproducibility in training new researchers.
